modules/clightning/Makefile-clightning

```diff
@@ -0,0 +1,1065 @@
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-strset.o: $(CCANDIR)/ccan/strset/strset.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-crypto-siphash24.o: $(CCANDIR)/ccan/crypto/siphash24/siphash24.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-htable.o: $(CCANDIR)/ccan/htable/htable.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-ilog.o: $(CCANDIR)/ccan/ilog/ilog.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-intmap.o: $(CCANDIR)/ccan/intmap/intmap.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-isaac.o: $(CCANDIR)/ccan/isaac/isaac.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-isaac64.o: $(CCANDIR)/ccan/isaac/isaac64.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-time.o: $(CCANDIR)/ccan/time/time.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-timer.o: $(CCANDIR)/ccan/timer/timer.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-io-io.o: $(CCANDIR)/ccan/io/io.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-io-poll.o: $(CCANDIR)/ccan/io/poll.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-io-fdpass.o: $(CCANDIR)/ccan/io/fdpass/fdpass.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-pipecmd.o: $(CCANDIR)/ccan/pipecmd/pipecmd.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-mem.o: $(CCANDIR)/ccan/mem/mem.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-fdpass.o: $(CCANDIR)/ccan/fdpass/fdpass.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-bitops.o: $(CCANDIR)/ccan/bitops/bitops.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-rbuf.o: $(CCANDIR)/ccan/rbuf/rbuf.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-str-base32.o: $(CCANDIR)/ccan/str/base32/base32.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-utf8.o: $(CCANDIR)/ccan/utf8/utf8.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-bitmap.o: $(CCANDIR)/ccan/bitmap/bitmap.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-membuf.o: $(CCANDIR)/ccan/membuf/membuf.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-json_escape.o: $(CCANDIR)/ccan/json_escape/json_escape.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-json_out.o: $(CCANDIR)/ccan/json_out/json_out.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-closefrom.o: $(CCANDIR)/ccan/closefrom/closefrom.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-rune-rune.o: $(CCANDIR)/ccan/rune/rune.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+ccan-rune-coding.o: $(CCANDIR)/ccan/rune/coding.c
+	@$(call VERBOSE, "cc $<", $(CC) $(CFLAGS) -c -o $@ $<)
+
+LIB_NAME = libclightning_wrapper.a
+
+CLIGHTNING_OBJS = \
+	clightning_lib.o
+
+$(LIB_NAME): $(CLIGHTNING_OBJS) libccan.a $(BITCOIN_OBJS) $(COMMON_OBJS) $(WIRE_OBJS) $(EXTERNAL_LIBS)
```

Comment:
Having a copy-paste of CLN's Makefile with this one addition is kinda ugly, and might break in the future.

It would be nice if we could just link against CLN instead.